src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Greg0ire\EnumBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class}
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('greg0ire_enum');
+
+        $rootNode->children()
+            ->enumNode('chain')
```

Comment:
This is not a chain, this is a format. A translation key format. Please name this parameter accordingly.